common/djangoapps/request_cache/middleware.py

```diff
@@ -1,3 +1,9 @@
+"""
+An implementation of a RequestCache. This cache is reset at the beginning
+and end of every request.
+"""
+
+import crum
```

Comment:
Mainly, I just wanted to have one place that was capturing the request, rather than 3 places. We were already using crum, so I figured I'd centralize on that.